src/Microsoft.ML.Core/Data/ColumnType.cs

```diff
@@ -554,43 +554,47 @@ public override bool Equals(ColumnType other)
     /// class numbers, etc. For example, in multi-class classification, the label is typically
     /// a class number which is naturally a KeyType.
     ///
-    /// KeyTypes can be contiguous (the class number example), in which case they can have
-    /// a cardinality/Count. For non-contiguous KeyTypes the Count property returns zero.
-    /// Any KeyType (contiguous or not) can have a Min value. The Min value is always >= 0.
+    /// KeyTypes with a known range of values have a cardinality/Count. If the range is
+    /// unknown the Count property returns zero.
```

Comment:
>unknown the Count property returns zero. [](start = 8, length = 40)

There is no longer any such concept. Key types always have known count in the new world.